modules/c3-assisted-installer-completing-installation-nodes.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-completing-installation-nodes_{context}"]
+= Assigning node roles
+
+If the Terraform scripts completed successfully, twelve hosts are now listed for the cluster. Three control plane hosts and three compute hosts have the status *Disconnected*. Three control plane hosts and three compute hosts have the status *Insufficient*. 
+
+Delete the disconnected hosts and assign roles to the remaining hosts. 
+
+.Procedure
+
+. From the link:https://console.redhat.com/openshift/assisted-installer/clusters[{ai-full} web console], select the cluster and navigate to the *Host discovery* page. 
+
+. Delete the six hosts with a *Disconnected* status, by clicking the option button for the host and selecting *Remove host*. The status of the remaining hosts changes from *Insufficient* to *Ready*. This can take up to three minutes.
+
+. From the *Role* column, assign the three nodes with a boot size of 1.10TB the *Control plane* role, and the three nodes with boot size 100GB the *Worker* role. 
+
+. Rename the hosts with a name shorter than 63 characters, by clicking the option button for the host and selecting *Change hostname*. Otherwise the cluster installation will fail. 
```

Comment:
```suggestion
. Rename any hosts with a name shorter than 63 characters by clicking the option button for the host and selecting *Change hostname*. Otherwise, the cluster installation will fail. 
```